Dockerfile.agnos

```diff
@@ -2,7 +2,7 @@
 # ###### Base ###### #
 # ################## #
 FROM scratch AS agnos-base
-ADD ubuntu-base-20.04.1-base-arm64.tar.gz /
+ADD ubuntu-base-24.04-base-arm64.tar.gz /
```

Comment:
Probably discussed somewhere already, but why dont we just use the Ubuntu docker as the base? As in "FROM ubuntu:24.04"

Reply:
We need to look over the base Dockerfiles - not sure why, to answer your question.

Reply:
Comparing
- https://cdimage.ubuntu.com/ubuntu-base/releases/24.04/release/ubuntu-base-24.04-base-arm64.tar.gz (what we use now)

with
- https://git.launchpad.net/cloud-images/+oci/ubuntu-base/tree/ubuntu-noble-oci-arm64-root.tar.gz?h=noble-24.04 (Docker base file e.g. "FROM ubuntu:24.04")

shows some differences, and I want to keep this PR stable

Reply:
Sounds good.